src/content/app/genome-browser/state/track-settings/trackSettingsSlice.ts

```diff
@@ -151,6 +151,8 @@ export const getTrackType = (trackId: string) => {
 
   if (trackId.startsWith('gene') || trackId === 'focus') {
     return TrackType.GENE;
+  } else if (trackId === 'focus-variant') {
```

Comment:
Can we use `TrackType.FOCUS_VARIANT` here?

Reply:
No. Track type is the type of a track. Track id is the identifier of the track. The fact that they are the same here is a coincidence.